app/controllers/application_controller.rb

```diff
@@ -1,5 +1,14 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  rescue_from CanCan::AccessDenied do |exception|
+    if current_user
+      render :file => "#{Rails.root}/public/403.html", :status => 403, :layout => false
```

Comment:
Maybe you should use actual erb views here to allow for a translation of the views.